src/components/Button.js

```diff
@@ -41,15 +43,13 @@ module.exports = React.createClass({
 		};
 	},
 	render() {
+		var classes = this.props.sheet.classes;
```

Comment:
There is truth in your words. However its a discussion not about jss, but whether to use or not to use decorators with react components. There has been lots of discussions about it, maybe @gaearon or @sebmarkbage can point us to them.